templates/documents/general_list.html

```diff
@@ -4,11 +4,14 @@
 {php}$this->assign('GLOBALS', $GLOBALS);{/php}
 
 <link rel="stylesheet" href="{$GLOBALS.css_header}" type="text/css">
+<link rel="stylesheet" href="{php}echo $GLOBALS['assets_static_relative']{/php}/dropzone-4-3-0/dist/dropzone.css">
```

Comment:
Considering Smarty is only used sporadically at the moment, let's hold off on adding it to places. I've been looking at [Twig as](https://twig.sensiolabs.org/) a stronger templating engine for use in the UI Modernization project. Let's just leave this as is for now. It's easy enough to fix later on. On an unrelated note, I'd like to see "assets_static_relative" renamed to "assets_dir" - shorter and simpler, though not a particularly important change. I updated a page recently and actually just did `$assets_dir = $GLOBALS['assets_static_relative'];` - it is a lot simpler when you're loading several assets